tap_xactly/streams.py

```diff
@@ -31,6 +31,7 @@ def sync(self):
         last_query_record_count = self.limit
         while last_query_record_count >= self.limit:
             try:
+                self.client.setup_connection()
```

Comment:
This causes the client to reconnect every time it queries the database. `setup_connection()` should be called only once unless the connection is closed for some reason.